server/src/test/java/org/spine3/server/bc/BoundedContextTestStubs.java

```diff
@@ -46,8 +45,8 @@ public static BoundedContext create(StorageFactory storageFactory) {
     }
 
     public static BoundedContext create(String name, StorageFactory storageFactory) {
-        final CommandBus commandBus = newCommandBus(storageFactory);
-        final EventBus eventBus = newEventBus(storageFactory);
+        final CommandBus commandBus = Given.Command.newCommandBus(storageFactory);
```

Comment:
Please have methods for commands creation only in `Command` classes. Move this one to `Given`.